### DISCUSSION TITLE
How to increase memory usage?

### DISCUSSION THREAD

**Discussion by thiagottjv** [madMAx43v3r/chia-plotter#270] [Q&A] [answered]
Hi guys, does anyone know how to increase the usage? 
I know it have something to do with the buckets, but should I increase or decrease it?
I have a R9 5900x and its using only around 17Gb of my 32Gb system (and waiting for another 32Gb to arrive).
Changing the bucket size would help in plots times? 
Is there a step number for the change? Like... 32, 64, 128, 256... ?

Thanks!

**Comment by SebMoore** [comment, marked as answer]
Decreasing buckets will increase RAM usage. Therefore, if you've got free ram, decrease the buckets. And yes - I believe there is a step - the default used to be 128, now being changed to 256. Try 64 buckets, and when your new 32g kit arrives, try 32 buckets.

**Reply by SebMoore** [reply]
It goes up/down in powers of 2, like you assumed (16,32,64,128,256,512 etc)

**Reply by thiagottjv** [reply]
Thanks, @VertiHydro, I`ll try that.
